charts/plugin-site/values.yaml

```diff
@@ -2,12 +2,46 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
 
-image:
-  repository: jenkinsciinfra/plugin-site
-  tag: 122-bd2a5f
-  pullPolicy: IfNotPresent
+backend:
+  replicaCount: 1
+  image:
+    repository: jenkinsciinfra/plugin-site-api
+    tag: 133-b6e652
+    pullPolicy: IfNotPresent
+  port: 8080
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
+    requests:
+      cpu: 1000m
+      memory: 1024Mi
+
+frontend:
+  replicaCount: 1
+  image:
+    repository: jenkinsciinfra/plugin-site
+    tag: 122-bd2a5f
+    oldMode: true
+    #repository: halkeye/jenkins-plugin-site
+    #tag: 44-14cfe6
+    pullPolicy: IfNotPresent
+  port: 80
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
+    requests:
+      cpu: 1000m
+      memory: 1024Mi
+  #resources:
+  #  limits:
+  #    cpu: 100m
```

Comment:
I assume these are the resources of you are using new mode?

Reply:
yea, and honestly probably a lot smaller than that, I don't fully understand the numbers :)

Reply:
This is pretty small I wouldn’t go smaller and should be fine for a static site

Reply:
```
$ kubectl top pod
NAME                                                      CPU(cores)   MEMORY(bytes)
jenkins-plugins-io-plugin-site-backend-6d7c96d98b-xvb4d   7m           489Mi
jenkins-plugins-io-plugin-site-frontend-9f6665996-ng799   1m           3Mi
```